service/awsconfig/v4/version_bundle.go

```diff
@@ -0,0 +1,66 @@
+package v4
+
+import (
+	"time"
+
+	"github.com/giantswarm/versionbundle"
+)
+
+func VersionBundle() versionbundle.Bundle {
```

Comment:
Since v4 it is nicer and we only have one version bundle for each resource version as it should be from here on.